components/list/list-item-checkbox-mixin.js

```diff
@@ -0,0 +1,123 @@
+import { css, html } from 'lit-element/lit-element.js';
+import { checkboxStyles } from '../inputs/input-checkbox-styles.js';
+import { getUniqueId } from '../../helpers/uniqueId.js';
+
+export const ListItemCheckboxMixin = superclass => class extends superclass {
+
+	/**
```

Comment:
Nit: let's get rid of the comments here.  We've been avoiding the comments except in cases where they are key to understanding why something is done where not obvious.

Reply:
I'm all for consistency for the most part, but I'd like to leave the function headers in. If they can help just one person while developing, I would say it's worth it. I have a plugin that shows function documentation automatically in my IDE, which I find pretty useful.

As an aside, there's also a plugin you can install that hides all comments and function headers by default :)

Reply:
Which IDE are you using.  Generally we've leaned toward keeping the comments lean since they are frequently redundant (as over the years we've gone down this path from time to time) and requires extra maintenance.  Happy to consider this, but would definitely like to be consistent throughout `core`.

Reply:
I use VScode. Function headers could save time in the long run despite extra maintenance - if someone is changing a function such that they need to change the function header, this can be a helpful sanity check. I'm always trying to think of ways to ease development for those who are new to a code base.

If this is something that we want to add to more of `core`, I'm happy to do that extra initial maintenance. Otherwise, I'm happy to keep a small island of functions with headers maintained where needed.

Reply:
I've removed function headers for now so we can discuss as a group another time :)